packages/itwinui-css/src/expandable-block/expandable-block.scss

```
@@ -2,219 +2,195 @@
+
+      cursor: not-allowed;
+      background-color: var(--_iui-expandable-block-header-background-color-disabled);
+      border-color: var(--iui-color-border-disabled);
+
+      @media (forced-colors: active) {
+        --_iui-expandable-block-expander-icon-fill: GrayText;
+        --_iui-expandable-block-header-caption-color: GrayText;
+        --_iui-expandable-block-header-label-color: GrayText;
+        border-color: GrayText;
+      }
     }
-  }
-}
 
-@mixin iui-expandable-block-header-label {
-  display: flex;
-  flex-direction: column;
-  flex: auto;
-  min-width: 0;
-  font-size: var(--iui-font-size-2);
-  color: var(--_iui-expandable-block-header-label-color, inherit);
-}
-
-/// @arg $type Can be one of: label, sublabel
-@mixin iui-expandable-block-header-label-text($type) {
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  transition: color var(--iui-duration-1) ease;
-  @media (forced-colors: active) {
-    transition: none;
-  }
-
-  @if $type == 'sublabel' {
-    font-size: var(--iui-font-size-0);
-    color: var(--_iui-expandable-block-header-caption-color, var(--iui-color-text-muted));
+    > .iui-expandable-block-icon {
+      @include mixins.iui-icon-style('m');
+      width: var(--_iui-expandable-block-expander-icon-size);
+      height: var(--_iui-expandable-block-expander-icon-size);
+      fill: var(--_iui-expandable-block-expander-icon-fill);
+      transform: var(--_iui-expandable-block-expander-icon-transform);
+      transition: fill var(--iui-duration-1) ease-out;
+      @media (prefers-reduced-motion: no-preference) {
+        transition: fill var(--iui-duration-1) ease-out, transform var(--iui-duration-1) ease-out;
+      }
+    }
   }
-}
-
-@mixin iui-expandable-block-content {
-  overflow: hidden;
-  background-color: var(--_iui-expandable-block-background-color);
-  border-bottom: var(--_iui-expandable-block-border);
-  border-right: var(--_iui-expandable-block-border);
-  border-left: var(--_iui-expandable-block-border);
-  border-bottom-left-radius: inherit;
-  border-bottom-right-radius: inherit;
-
-  @include mixins.iui-transition-group;
 
-  // padding should be in a separate div because otherwise it messes up height transition
-  > div {
-    padding: var(--_iui-expandable-block-content-padding);
+  .iui-expandable-content {
+    background-color: var(--_iui-expandable-block-background-color);
+    border-right: var(--_iui-expandable-block-border);
+    border-left: var(--_iui-expandable-block-border);
+    border-bottom-left-radius: inherit;
+    border-bottom-right-radius: inherit;
+
+    display: grid;
+    grid-template-rows: 0fr;
+    overflow: hidden;
+    z-index: -1;
+    transition: grid-template-rows 0.2s, visibility 0.2s, transform 0.2s;
+
+    > div {
+      min-height: 0;
+      visibility: hidden;
+      padding-inline: var(--_iui-expandable-block-content-padding);
+      transition: visibility 0.2s ease-out;
+      align-self: start;
```

this might be subjective but it looks slightly better with `end` (could be an illusion though)

```suggestion
      align-self: end;
```